package.json

```
@@ -12,7 +12,8 @@
     "copy-views": "mkdir -p dist/views && cp src/views/*.njk dist/views",
     "build": "npm run clean && npm run build-sass && npm run copy-assets && npm run copy-views && tsc",
     "test": "npm run lint && mocha src/test",
-    "start": "node ."
+    "start": "node .",
+    "dev": "nodemon ."
```

```suggestion
    "dev": "npm run build && nodemon ."
```
I think we should run the build step first - when I checked out this branch and ran `npm run dev` I got a bunch of errors because I hadn't built since switching from a different branch

---

thanks for the suggestion Alex, have updated